tests/api/controllers/test_credential.py

```diff
@@ -35,4 +35,3 @@ def test_update_credential(self, mock_vault_encrypt):
                                 content_type='application/json')
         self.assertTrue(mock_vault_encrypt.called)
         mock_vault_encrypt.assert_called_with(value)
-        self.assertEqual(http_client.NO_CONTENT, response.status_code)
```

Comment:
think you can add back an 
self.assertEqual(http_client.OK, response.status_code)